mir-ci/mir_ci/benchmarker.py

```diff
@@ -0,0 +1,91 @@
+import asyncio
+import time
+import logging
+from typing import Dict, Callable, Tuple, List, Optional
+from abc import ABC, abstractmethod
+from contextlib import suppress
+
+from mir_ci.cgroups import Cgroup
+from mir_ci.interfaces.benchmarkable import Benchmarkable
+from mir_ci.interfaces.benchmarker_backend import BenchmarkBackend
```

Comment:
Unused imports (will add some linting to CI soon), more below, too:

```suggestion
import asyncio
import logging
from typing import Dict, Callable, Optional
from contextlib import suppress

from mir_ci.interfaces.benchmarkable import Benchmarkable
from mir_ci.interfaces.benchmarker_backend import BenchmarkBackend
```

Reply:
I can `autoflake` it for the time being :)